snippets/legv8.json

```diff
@@ -0,0 +1,185 @@
+{
+  "set register to": {
+    "prefix": "sreg",
+    "body": ["ADD ${1:dest reg}, XZR, $2", "$3"]
+  },
+  "increment register": {
+    "prefix": "increg",
```

Comment:
Since it is only incrementing by one, would a prefix of `reg++` or `inc_1` make more sense?

Reply:
I removed that snippet also